<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderReportsWidget</class>
 <widget class="QWidget" name="OrderReportsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>921</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OrderReportsWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBoxMain">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="buttonImportReports">
          <property name="text">
           <string>Importer un ou plusieurs rapports</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonHelp">
          <property name="text">
           <string>?</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonLoadDataExisting">
          <property name="toolTip">
           <string>Charger les rapports sélectionnés</string>
          </property>
          <property name="text">
           <string>Charger</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonLoadAll">
          <property name="toolTip">
           <string>Charger tous les rapports</string>
          </property>
          <property name="text">
           <string>Charger tout</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonRemoveReports">
          <property name="toolTip">
           <string>Supprimer les rapports sélectionnés</string>
          </property>
          <property name="text">
           <string>Supprimer</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="styleSheet">
         <string notr="true">QSplitter::handle {background: black;}</string>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QTreeView" name="treeViewFolder">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
        </widget>
        <widget class="QTabWidget" name="tabWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>3</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tabChart">
          <attribute name="title">
           <string>Graphique</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Nombre de commandes par jour (pour vérifier qu'aucun fichier n'a été oublié)</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QtCharts::QChartView" name="chartView">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>4</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="WidgetOrders" name="tabOrders">
          <attribute name="title">
           <string>Commandes</string>
          </attribute>
         </widget>
         <widget class="WidgetRefunds" name="tabRefunds">
          <attribute name="title">
           <string>Remboursements</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4"/>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
  <customwidget>
   <class>WidgetOrders</class>
   <extends>QWidget</extends>
   <header>gui/panes/widgets/WidgetOrders.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetRefunds</class>
   <extends>QWidget</extends>
   <header>gui/panes/widgets/WidgetRefunds.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
